components/brave_new_tab_ui/stories/fakeLocale.ts

```diff
@@ -4,8 +4,7 @@
 
 const locale: any = {
   // stats
-  trackersBlocked: 'Trackers Blocked',
-  adsBlocked: 'Ads Blocked',
+  adsBlocked: '“Ads and Trackers Blocked',
```

Comment:
Is that a stray double quote character in that string?

Reply:
It is indeed. It's test data so non visible but it shouldn't be there. Good eye. Fixed.